### PR TITLE
random: Add descriptions to Randomizer::shuffleArray() and Randomizer::shuffleBytes()

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

File: reference/random/random/randomizer/shufflearray.xml [hunk 1]
```diff
@@ -12,11 +12,17 @@
...
   </para>
 
-  &warn.undocumented.func;
+  <para>
+   Each possible permutation of the input <parameter>array</parameter> is equally likely to be returned.
+  </para>
 
+  <para>
+   The input <parameter>array</parameter> will not be modified. Array keys of the input <parameter>array</parameter>
+   will not be preserved; the returned &array; will be a list (<function>array_is_list</function>).
```
Comment (cmb69) [inline reference/random/random/randomizer/shufflearray.xml:24]: Shouldn't we move the second sentence to the return-values section?

**Review by cmb69 — APPROVED** [review]
Thank you!

File: reference/random/random/randomizer/shufflearray.xml (hunk 1 above)
Comment (Girgias) [reply reference/random/random/randomizer/shufflearray.xml:24]: I agree that it makes more sense to have the second sentence in the return value section, and maybe the first one in the parameter description

Comment (TimWolla) [reply reference/random/random/randomizer/shufflearray.xml]: Okay, updated, please have another look. I've also fixed the texts properly refer to `shuffleString()`'s parameter as `$bytes`, not `$string`.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Review by Girgias — APPROVED** [review]

**Review by zeriyoshi — APPROVED** [review]

**Pull request merged**